### DISCUSSION TITLE
Got the T Dongle S3 and can't figure out how to set it up

### DISCUSSION THREAD

**Discussion by GotterDameron** [BitMaker-hub/NerdMiner_v2#436] [General]
Could someone give some help setting up my Dongle, please?

I can't even find an AP configuration and it talks about phone settings when I don't access the Internet by phone.

If anyone knows what I should do here, could you give me a hand?

**Comment by BitMaker-hub** [comment]
Hi Gotter, It should be allways the same on all boards.

You have to access to an access point (a Wifi called NerdMinerAP). 
Connect to it from your mobile phone and use the password MineYourCoins

Then, you will access to a config website and follow configurations. 

check my YT videos or bitronics guide: https://bitronics.store/storage/2023/07/NM2manual.pdf

**Reply by GotterDameron** [reply]
Thanks!
